### DISCUSSION TITLE
Add option to give permanent trust or join multiple island

### DISCUSSION THREAD

**Discussion by SkyDude0** [BG-Software-LLC/SuperiorSkyblock2#1958] [Approved Feature Requests]
HI,
Since it is not possible to be part of more than 1 island, I propose that at least trusts have the option of being permanent.
Although the option of being part of multiple islands is better.

**Comment by OmerBenGera** [comment]
You can do that with the `/is permissions` system.

**Comment by SkyDude0** [comment]
No, i mean, if i trust someone and the player left the server. When he come back, he dosent have the trust anymore.

I suggest atleast this, since player can only be part on max 1 island

**Comment by Tuogptuan** [comment]
When the player quit the server, he will be uncooped from all the islands that have cooped the player, and he does not want to.

**Comment by OmerBenGera** [comment]
As I said, you can use `/is permissions <player-name>` to give permissions to other players to certain actions on your island, that will be permanent.

**Comment by Tuogptuan** [comment]
It would be simpler if there was an option to turn it on or off instead of adding each coop's role permission to the player.
